perf/istio-install/gke_cluster_resources/cluster.yaml

```diff
@@ -0,0 +1,54 @@
+# Copyright Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: identity.cnrm.cloud.google.com/v1alpha2
+kind: IdentityNamespace
+metadata:
+  name: default
+  namespace: "test-istio-237517" # {"type":"string","x-kustomize":{"setter":{"name":"gcloud.core.project","value":"test-istio-237517"}}}
+spec: {}
+---
+apiVersion: container.cnrm.cloud.google.com/v1alpha2
+kind: ContainerCluster
+metadata:
+  clusterName: "test-cluster3-istio-237517/us-central1-a/test-cluster3" # {"type":"string","x-kustomize":{"partialSetters":[{"name":"gcloud.core.project","value":"test-istio-237517"},{"name":"cluster-name","value":"test-cluster3"},{"name":"gcloud.compute.zone","value":"us-central1-a"}]}}
+  name: test-cluster3 # {"type":"string","x-kustomize":{"setter":{"name":"cluster-name","value":"test-cluster3"}}}
+  namespace: "test-istio-237517" # {"type":"string","x-kustomize":{"setter":{"name":"gcloud.core.project","value":"test-istio-237517"}}}
+spec:
+  location: us-central1-a # {"type":"string","x-kustomize":{"setter":{"name":"gcloud.compute.zone","value":"us-central1-a"}}}
+  workloadIdentity:
+    identityNamespace: default
+  labels:
+    mesh_id: "proj-920942343357" # {"type":"string","x-kustomize":{"partialSetters":[{"name":"gcloud.project.projectNumber","value":"920942343357"}]}}
```

Comment:
this should be `proj-[PROJECT_NUMBER]` or some other placeholder

Reply:
done